features/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/themes/default/js/logic/flot.stat.js

```diff
@@ -94,7 +92,7 @@ var onDateSelected = function(from, to) {
 			'isOnChoice' : true
 		},
 		success : function(response) {
-			var parsedResponse = JSON.parse(response);
+			var parsedResponse = response;
```

Comment:
removed redundant variable 'parsedResponse' in commit dd21bc94c72b34390429b6365774638e78bc7f56 of PR #240